amgut/lib/data_access/test/test_ag_data_access.py

```diff
@@ -426,19 +423,51 @@ def test_getAvailableBarcodesNotPresent(self):
         self.assertEqual(res, [])
 
     def test_verifyKit(self):
-        raise NotImplementedError()
+        # Test verifying works
+        kit = self.ag_data._get_unverified_kits()[0]
+        self.ag_data.verifyKit(kit)
+        obs = self.ag_data.getAGKitDetails(kit)
+        self.assertEqual(obs['kit_verified'], 'y')
 
-    def test_getMapMarkers(self):
-        raise NotImplementedError()
+        # Test verifying a non-existant kit
+        # TODO: make raise an error
+        self.ag_data.getAGKitDetails('NOTAREALKITID')
```

Comment:
Why is this here?  Even if it doesn't raise an error, should there be some sort of check?